source/runner/templates/github-issue-template.ts

```diff
@@ -11,7 +11,7 @@ import * as v from "voca"
  * @returns {string} HTML
  */
 function table(name: string, emoji: string, violations: Array<Violation>): string {
-  if (violations.length === 0) { return "" }
+  if (violations.length === 0 || (violations.length === 1 && !violations[0].message)) { return "" }
```

Comment:
Lets get a test on this logic, as this is not something which would get manually tested very often